src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -439,9 +443,13 @@
           <div *ngIf="hasCompressionEnabled()">
             <!-- Compression algorithm selection -->
             <div class="form-group row">
-              <label i18n
-                     class="cd-col-form-label"
-                     for="algorithm">Algorithm</label>
+              <label class="cd-col-form-label"
+                     for="algorithm">
+                <ng-container i18n>Algorithm</ng-container>
+                <cd-helper>
+                  <span i18n>Compression algorithm used for BlueStore</span>
```

Comment:
```suggestion
                  <span i18n>Compression algorithm used </span>
```
I don't think the user should know about BlueStore

Reply:
I agree, Lets not use any non generic terms to explain to user.